app/admin/news_article.rb

```diff
@@ -0,0 +1,51 @@
+if defined?(ActiveAdmin)
+  ActiveAdmin.register News::Article, as: 'News' do
+    index do
+      selectable_column
+      column :id
+      column :title
+      column :annonce
+      actions
+    end
+
+    show do
+      attributes_table do
+        row :id
+        row :title
+        row (:text)  { |article| raw(article.text) }
+        row :text
+      end
+    end
+
+    form do |f|
```

Comment:
Unused block argument - `f`. You can omit the argument if you don't care about it.